Demo/SnapshotTests/ComponentViewTests.swift

```diff
@@ -294,4 +294,8 @@ class ComponentViewTests: XCTestCase {
     func testDetailCallout() {
         snapshot(.detailCallout)
     }
+
+    func testMonthAndYearPickerView() {
+        snapshot(.monthAndYearPickerView)
```

Comment:
The snapshot image looks spicy😄 🌶️ It seems like it could fail later if it lays out properly before the test is run, so maybe we should we introduce a small delay to this test, just in case it should fail inconsistently?

![Screenshot 2022-05-02 at 15 42 13](https://user-images.githubusercontent.com/1901556/166243791-a21ae568-f94f-4051-bc27-3e6c4fd70126.png)

Reply:
Yes, good point. I'll look into it :)

Reply:
No delay seems to work, tried multiple different time intervals, the snapshot still always look the same 🤷‍♂️